### DISCUSSION TITLE
Einstellung "Nulle Tagesertrag um Mitternacht" verloren?

### DISCUSSION THREAD

**Discussion by huste511** [tbnobody/OpenDTU#1681] [General]
Moin!

Seit dem es die Option "Tagesertragskorrektur" gibt, hatte ich ebenso die Option "Nulle Tagesertrag um Mitternacht" aktivert.
Bin ich mit relativ sicher, habe aber leider kein config-Backup aus der Zeit. ;-(

Ziel war es eben ein /einheitliches/ Null um Mitternacht zu haben und nicht irgendwelche Teilwerte je nach Laune der WR.

Die letzten Wochen habe ich jeweils die neuen Versionen eingespielt und nun habe ich festgestellt, dass die Anzeige Nachts eben nicht immer auf Null wechselt und bei allen WR die Einstellung nicht mehr aktivert ist.

Hat das noch jemand festgestellt oder spielt mit das Hirn einen Steich?

**Comment by MetaChuh** [comment]
bisher nicht festgestellt.

anmerkung: falls du nachts einen inverter mit batterie betreibst, musst du diesen um 23:59 per befehl via opendtu neu starten (bitte inverter neustart, nicht dtu neustart)

sonst setzt die dtu zwar intern den tagesertrag zurück, jedoch bei der nächsten abfrage, wird der alte tagesertrag aus dem hm inverter wieder eingelesen, der solange hochzählt, bis entweder der inverter dc stromlos ist, oder rebootet wird.
